webpack.renderer.ts

```diff
@@ -87,6 +88,13 @@ export function webpackLensRenderer({ showVars = true } = {}): webpack.Configura
     },
     module: {
       rules: [
+        {
+          // Allow to load raw file contents with dynamic imports, e.g. import("../some-path.txt?raw")
+          // See also: https://webpack.js.org/guides/asset-modules/
+          use: "raw-loader",
+          resourceQuery: /raw/,
+          enforce: "post",
+        },
```

Comment:
I don't think we use this in this PR.

Reply:
This is fine.

Reply:
I don't understand your frenetic desire to restrict merging _every single line of code_  if non-relevant to PR/topic..
Why so strict? What the purpose? Clear history? Blah.. It doesn't help with bugs / don't provide any benefit to end  users..

Reply:
I agree with @Nokel81 that we should not do include in any changes that are not necessary for the PR. Making separate PR will make it faster to get these changes in. If this PR does not work without this change, then this change should belong to this PR.

Reply:
Thanks for your opinions. Removed.